src/BackwardCompatiblityTests/Version_0_2_0.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Axe.Windows.Actions;
+using Axe.Windows.Actions.Contexts;
+using Axe.Windows.Core.Bases;
+using Axe.Windows.Core.Enums;
+using Axe.Windows.Core.Results;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Axe.Windows.BackwardCompatiblityTests
```

Comment:
```suggestion
namespace Axe.Windows.BackwardCompatibilityTests
```

Reply:
Renamed to Axe.Windows.OldFileVersionCompatibilityTests